devtools-ci-index.yaml

```diff
@@ -3165,6 +3165,11 @@
             ci_cmd: '/bin/bash cico_build.sh'
             timeout: '20m'
             discarder_days: 30
+        - '{ci_project}-{git_repo}-build-master-coverage':
```

Comment:
Well... -build-master also sets up secrets for quay and devshift which we don't need because we don't deploy anything. We just build. Those secrets wouldn't break anything though..
I see a few options here:
1. Create a new job template -build-mater-<something?> which would be a copy of -build-master-coverage
2. Rename -build-master-coverage to something more generic. There is nothing test coverage specific in this template. It's just a build without deployments.
3. Use the existing -build-master (with unused secrets).
4. Use the existing -build-master-coverage (which can be confusing).

WDYT?

Reply:
I would go with first. job-template - `-build-master-update-downstream`, After copying `build-master-coverage`, I would also add `FABRIC8_HUB_TOKEN` as a secret to it to push an updated client to its consumer which we'll need as a next step. Ref https://github.com/openshiftio/openshiftio-cico-jobs/pull/861/files#diff-a0fd432ddaac45438209cf6e0a7298c0R1485

Reply:
@dipak-pawar done

Reply:
thanks @alexeykazakov